workflow/scripts/predict.py

```diff
@@ -86,8 +86,8 @@ def main():
     enhancers = enhancers_full.loc[:,['chr','start','end','name','class','activity_base','normalized_dhs' ]]
     enhancers['activity_base_squared'] = enhancers['activity_base']**2
     #Initialize Prediction files
-    pred_file_full = os.path.join(args.outdir, "EnhancerPredictionsFull.txt")
-    pred_file_slim = os.path.join(args.outdir, "EnhancerPredictions.txt")
+    pred_file_full = os.path.join(args.outdir, "EnhancerPredictionsFull.csv")
```

Comment:
I noticed these files are really csv files. Since other code depends on these being csv files (grabMetrics.py), we should label it appropriately?